CI test on Upmerge22073 (#22148)
nrfconnect/sdk-nrf: applications/nrf_desktop/doc/cpu_meas.rst

*************

To enable this module, use the :ref:`CONFIG_DESKTOP_CPU_MEAS_ENABLE <config_desktop_app_options>` Kconfig option.

> This option selects the :kconfig:option:`CONFIG_NRF_CPU_LOAD` option.
> The :kconfig:option:`CONFIG_NRF_CPU_LOAD` option enables the :ref:`cpu_load` library that is used to perform the measurements.

Reviewer: ```suggestion
This option selects the :kconfig:option:`CONFIG_NRF_CPU_LOAD` option that enables the :ref:`cpu_load` library used for performing the measurements.
```

Author: thanks but that's just a CI test. No need for reviews here.

Reviewer 2: ?
